api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -336,6 +340,7 @@ private void validateWorkspaceTierAccess(DbWorkspace dbWorkspace) {
 
     if (!workspaceIdsToDelete.isEmpty()) {
       userRecentWorkspaceDao.deleteByUserIdAndWorkspaceIdIn(userId, workspaceIdsToDelete);
+      userRecentlyModifiedResourceDao.deleteByUserIdAndWorkspaceIdIn(userId, workspaceIdsToDelete);
```

Comment:
this is probably a good idea in the future, but if we implement this now, it would cause a discrepancy between the old table and the new table.